File: task-03/prime.rb
def is_prime(num)
    return false if num <= 1

    (2..Math.sqrt(num)).each do |i|
        return false if num % i == 0
    end

    return true
end

puts "Enter any number: "
n = gets.chomp.to_i

(2..n).each do |i|
    if is_prime(i)
        print i, " "
    end
end

File: task-03/prime.go
package main

import "fmt"

func isPrime(num int) bool {
    if num <= 1 {
        return false
    }

    for i := 2; i < num; i++ {
        if num%i == 0 {
            return false
        }
    }

    return true
}

func main() {
    var n int
    fmt.Print("Enter any number: ")
    fmt.Scan(&n)

    for i := 2; i <= n; i++ {
        if isPrime(i) {
            fmt.Printf("%d ", i)
        }
    }
}
